src/main/g8/default.properties

```diff
@@ -16,4 +16,4 @@
 name=format
 extension=dat
 package=com.example
-namespaced=yes
+namespaced=no
```

Comment:
Don't we still want things namespaced to avoid possible conflicts with file names? I thought this should only be set to know for testing/example projects?

Reply:
I changed it to 'no' because that's what the README.md already said. 

I'm happy with 'no' as the default, as I expect few people to have patience with the deeper package-like paths.